### ISSUE TITLE
Headline-tree is empty in Outline-View

### ISSUE THREAD

**Issue by r-pf** [de-jcup/eclipse-asciidoctor-editor#381]
Preferences: 
In the preferences for Asciidoctor Editor I set the value for "Default preview layout" to "Hide preview, use external browser".

Action:
When opening an asciidoc-file the Outline-View remains empty, even when the asciidoc-file contains headlines.
After changing the value of "Default preview layout" to an other value the headline-tree appears in the Outline-View.
... and then changing back to "Hide preview, use external browser" the headline-tree keeps alive (it's magic ;-) ).

The empty Outline-View appears after starting eclipse for all asciidoc-file opened automatic because they were open when closing eclipse.
After performing the pseudo-change mentioned above the headline-tree appears in all afterwards opened asciidoc-files, but not for the asciidoc-files which were already open when performing the pseudo change.

This behaviour appears in version 2.4.1 (and not in version 2.2.0).
I tested 2.4.1 with eclipse 2021-06, 2018-12 and 2.2.0 with eclipse 2018-12

**Comment by de-jcup** [comment]
## Reproduce with eclipse 2021-12
1. Used eclipse-2021-12 + current development state of plugin
2. Created file `test-on-startup.adoc` with two headlines inside 
3. Execute "File->Restart"
4. Now the outline is initial empty (see next figure)
![image](https://user-images.githubusercontent.com/11644753/158953261-2e157688-ae66-4276-8df3-82e501c8f948.png)

## Reproduce with eclipse 2018-x
Not reproducable - exactly as explained in issue.

## Check F5/refresh action
- Press `F5` -> model is built and outline as expected:
![image](https://user-images.githubusercontent.com/11644753/158953799-1e1bea07-438e-49ee-a364-675755b7c9f1.png)

## Check changes on editor content
- Restart again
- clean errror log view
- change content
- press `CTRL S` to save editor - exeption happens:
 ![image](https://user-images.githubusercontent.com/11644753/158954233-25363372-f68e-4983-a525-962e51de7f1d.png)
- Inspected exception:
```
java.lang.NullPointerException: Cannot invoke "de.jcup.asciidoctoreditor.asciidoc.AsiidocConfigFileSupport.calculateResolvedMap(java.util.Map, java.util.List)" because "support" is null
	at de.jcup.asciidoctoreditor.provider.AsciiDoctorAttributesProvider.resolveAttributes(AsciiDoctorAttributesProvider.java:106)
	at de.jcup.asciidoctoreditor.provider.AsciiDoctorAttributesProvider.getCachedAttributes(AsciiDoctorAttributesProvider.java:94)
	at de.jcup.asciidoctoreditor.provider.AsciiDoctorImageProvider.resolveImagesDirPath(AsciiDoctorImageProvider.java:138)
	at de.jcup.asciidoctoreditor.provider.AsciiDoctorImageProvider.getCachedSourceImagesPath(AsciiDoctorImageProvider.java:92)
	at de.jcup.asciidoctoreditor.AsciiDoctorEditor.getImagesPathOrNull(AsciiDoctorEditor.java:520)
	at de.jcup.asciidoctoreditor.AsciidoctorEditorOutlineSupport.validate(AsciidoctorEditorOutlineSupport.java:178)
	at de.jcup.asciidoctoreditor.AsciidoctorEditorOutlineSupport.lambda$0(AsciidoctorEditorOutlineSupport.java:139)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:40)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:185)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:5101)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:4584)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1154)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1045)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:644)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:551)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:156)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:152)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:401)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:659)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:596)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1467)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1440)
```

The exception also happens 
- directly after every restart of eclipse with given setup
- on every change while no refresh action has been triggered
- also closing + opening same file again brings up same exception
- pressing F5 does bring up outline again . Then close and open file again: outline is there and the exception does no longer appear when changing + saving the document.

**Comment by de-jcup** [comment]
This is related to
- https://github.com/de-jcup/eclipse-asciidoctor-editor/issues/385
- https://github.com/de-jcup/eclipse-asciidoctor-editor/issues/370

**Comment by de-jcup** [comment]
I think this is already solved with #385
